dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/OidcAuthorizationRequest.java

```diff
@@ -226,9 +226,18 @@ void addOptionalParameters(AuthorizationRequestParameters authzParameters, OidcC
             addPkceParameters(pkceCodeChallengeMethod, state, authzParameters);
         }
 
+        // LIBERTY-I-61: START
         if (isImplicit) {
             addImplicitParameters(authzParameters);
         }
+        String resources = getResourcesParameter();
+        System.out.println("Resources: " + resources);
```

Comment:
`System.out` statements shouldn't be used in the code. `Tr.debug()` should be used instead for things that will make debugging easier, like so:
```Java
        if (tc.isDebugEnabled()) {
            Tr.debug(tc, "Resources: " + resources);
        }
```
There's also no need to log just the `resources` variable without any other descriptive information because Liberty will automatically log entry and exit statements for every method that is called, along with the arguments to the methods and the returned value from the method, like so:
```
[9/19/23, 10:04:33:511 CDT] 0000003d id=18fbd8ba com.ibm.ws.security.authentication.WSAuthenticationData      > get Entry  
                                                                                                               CERTCHAIN
[9/19/23, 10:04:33:511 CDT] 0000003d id=18fbd8ba com.ibm.ws.security.authentication.WSAuthenticationData      < get Exit  
                                                                                                               null
```
That means calling the `getResourcesParameter()` method above will already be logged automatically if the customer has trace enabled, so they'd see something like
```
[9/19/23, 10:04:33:511 CDT] 0000003d id=18fbd8ba com.ibm.ws.security.openidconnect.clients.common.OidcAuthorizationRequest > getResourcesParameter Entry  
[9/19/23, 10:04:33:511 CDT] 0000003d id=18fbd8ba com.ibm.ws.security.openidconnect.clients.common.OidcAuthorizationRequest < getResourcesParameter Exit  
                                                                                                               http://localhost/resource1, http://localhost/resource2
```
Having a separate trace statement just to log the `resources` variable would be unnecessary.